smartmeter/images.h

```diff
@@ -0,0 +1,95 @@
+#define WiFi_Logo_width 60
```

Comment:
Add a `#pragma once` on the beginning of the file, like display.h
This is used to result in a faster compilation time, the file will only be included by the compiler memory only one time.

Reply:
Done